src/common/logger.ts

```diff
@@ -1,6 +1,6 @@
 import debug from "debug";
 
-if (!process.env.JEST_WORKER_ID) debug.enable("*"); // enable log outputs if not running in jest
+debug.enable("api-notarize-healthcerts:*,transient-storage-lib:*");
```

Comment:
we also have other packages log like gpay, eu-dcc packages etc. so, can u keep it as `*` like before ?